src/main/java/com/example/backendspringboottechiteasycontroller/dtos/TelevisionInputDTO.java

```diff
@@ -0,0 +1,189 @@
+package com.example.backendspringboottechiteasycontroller.dtos;
+
+import com.example.backendspringboottechiteasycontroller.models.TVOption;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TelevisionInputDTO {
+    private Integer id;
```

Comment:
The input DTO doesn't need an ID, because it will be generated by the `@`GeneratedValue annotation in the Television class. Even in a PUT request, you don't need the ID in the input DTO, because it will be given as a PathVariable

Reply:
Thank you Mark for your constructive feedback :D. I have a question here because I agree with your point of view but isn't in a PUT request and the ID is passed to through a PathVariable will I need to assign it to my DTO and then through the mapper pass it on to my model ? or assign the ID (PathVariable) to the mapper / or maybe even the model immediately ? I'm confused about that?  what do you think is the best here ?

Reply:
If I understand you wuestion correctly, there's different approaches to solve this problem. I'll try to show one of them in the huiswerkklas tommorrow. It is to overwrite the mapper function so that you can add an existing television in stead of letting the function make a new television by default. That way you can let the mapper use the existing television to tranfer the values to.
Another approach is to simply do it in the service method.

Reply:
My solution here was to for the 3rd time override the method mapTelevisionDTOToTelevision(/*2 parameters*/ TelevisionInputDTO , Integer id) will push that here on this branch before pushing to the main. I also would prefer as Frans mentioned to keep the mapping all in all in the controller layer